### DISCUSSION TITLE
How should Maintainers be picked?

### DISCUSSION THREAD

**Discussion by Philip-Clark** [club-40/odin-pantry#4] [Polls]
We need to decide how to select maintainers, please give your input with the poll below.
Thanks!

**Comment by RyanMcEntire** [comment]
my option 4 would be to have the maintainers primarily have experience, but also have people who express interest who would like to learn and get practice, who already know the technologies.

as for voting. i don't know enough about everybody to vote for maintainers. i don't think i'm alone in that. i say just pick the people who have the most experience (and want to be maintaner) and have those people pick some newer people that would like the experience. i don't favor pure majority democracy at this step.

**Comment by ManorSailor** [comment]
I agree with @RyanMcEntire above. Most of the individual that are collaborating on this project aren't (likely) experienced enough to be maintainers. Better option is to have at least one experienced maintainer for the project alongside some other folks who chooses to assist the maintainer(s).

**Reply by Aureatus** [reply]
I'd probably go for an experienced person or two to be maintainers for both ends of the project, depending on the amount of people each area has.

**Comment by Zadag** [comment]
KB here.  I agree with Ryan and Sars.  We don't know enough about each other to make an informed vote and there is likely a lack of experience among those voting.  If somebody has experience either through TOP or their job with project management or collaborative development and they are interested in heading the project they would be the clear choice; no need for a vote.  If nobody fits those qualifications I would defer to either BiFF, or a willing TOP maintainer to choose 1 or 2 people.

**Reply by Philip-Clark** [reply]
I would defer to a TOP mod or maintainer if possible.

**Comment by Philip-Clark** [comment]
From the few votes we have so far, I think finding an experienced member of the TOP team is something we all would like. I'm not sure the poll had the proper options to express that, sorry about that.

If you are such a member of the TOP team, we invite you to this role if it interests you. Thanks
